CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+### 2.2.3
+
+- Fix iOS picker's styling bug where picker would be fixed to 216px height no matter what `style` was applied to the picker. Now it can correctly apply specific height settings and flex. NOTE: this only works with View style property type. [#120](https://github.com/react-native-community/react-native-datetimepicker/pull/120)
```

Comment:
```suggestion
- Fix iOS picker's styling bug where picker would be fixed to 216px height no matter what `style` was applied to the picker. This is because previously, the picker was always wrapped in a `View` which was now removed. Since `style` prop was never documented in this package, we do not consider this a breaking change. Now `style` can correctly apply specific height settings and flex. NOTE: this only works with View style property type. [#120](https://github.com/react-native-community/react-native-datetimepicker/pull/120)
```